content/en/docs/setup/minikube.md

```diff
@@ -9,146 +9,206 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs a single-node Kubernetes cluster inside a VM on your laptop for users looking to try out Kubernetes or develop with it day-to-day.
+Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs a single-node Kubernetes cluster inside a Virtual Machine (VM) on your laptop for users looking to try out Kubernetes or develop with it day-to-day.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
 ## Minikube Features
 
-* Minikube supports Kubernetes features such as:
-  * DNS
-  * NodePorts
-  * ConfigMaps and Secrets
-  * Dashboards
-  * Container Runtime: Docker, [rkt](https://github.com/rkt/rkt), [CRI-O](https://github.com/kubernetes-incubator/cri-o) and [containerd](https://github.com/containerd/containerd)
-  * Enabling CNI (Container Network Interface)
-  * Ingress
+Minikube supports the following Kubernetes features:
+
+* DNS
+* NodePorts
+* ConfigMaps and Secrets
+* Dashboards
+* Container Runtime: Docker, [rkt](https://github.com/rkt/rkt), [CRI-O](https://github.com/kubernetes-incubator/cri-o), and [containerd](https://github.com/containerd/containerd)
```

Comment:
These links could be glossary entries instead. Does that sound worthwhile?

Something like this:
```suggestion
* Container Runtime: {{< glossary_tooltip text="Docker" term_id="docker" >}}, {{< glossary_tooltip  term_id="rkt" >}}, {{< glossary_tooltip text="CRI-O" term_id="cri-o" >}}, and {{< glossary_tooltip  term_id="containerd" >}}
```

Docker needs PR #14280 as the existing entry isn't appearing. I can add glossary entries for CRI-O and containerd, if that's helpful &mdash; they're not in the glossary at the moment.

Reply:
That's a great idea! :) +1 .

Reply:
Hi @sftim,
 We can update the page with this change after this PR gets merged? Or is there a way to do both at once?

Reply:
I don't think Prow does [octopus merges](https://git-scm.com/docs/merge-strategies#Documentation/merge-strategies.txt-octopus) just yet :grin:

I think it's fine to put those tool tips / glossary references in after #14309 gets merged, and that shouldn't block this PR from landing.